cmd/transflect-operator/operator.go

```diff
@@ -47,10 +51,11 @@ type operator struct {
 	// so that no two updates for a single deployment can run concurrently.
 	deploymentLocker transflect.MutexMap
 
-	useIngress bool
-	plaintext  bool
-	address    string
-	version    string
+	useIngress      bool
+	plaintext       bool
+	address         string
+	version         string
+	leaderNamespace string
```

Comment:
Just wondering if this could be `leaseNamespace` for consistency?

Reply:
done, thanks!